README.md

```diff
@@ -26,6 +26,18 @@ source <path/to/.venv>/bin/activate
 3. Build OCP4MCO-CI: `python setup.py  build`
 3. Install OCP4MCO-CI: `python setup.py  install`
 
+## Configuration
```

Comment:
This configuration is required for someone who wants to execute this script as a scheduler. In case, I want to deploy the cluster now, I can use the deploy command directly. 
 
 So this configuration part has to mentioned as optional, and only for scheduled cluster deployment

Reply:
Done.